index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+        rel="stylesheet">
+    <link rel="stylesheet" href="index.css">
+    <title>mobile skeleton</title>
+</head>
+<body>
+<main>
+    <div class="full-wrapper">
+        
+        <nav>
+            <div class="imgs">
+                <img src="/images/Light.png" alt="">
+            </div>
+            <div class="nav-wrapp">
+                <div class="logo">
+                    <span>My Logo</span>
+                </div>
+                <div class="right-nav">
+                    <i id="menu" class="material-icons">menu</i>
+                </div>
+            </div>
+        </nav>
+    
+        <div class="content-wrapper">
+            <div class="header">
+                <span>I’m Desmond
+                    Glad to see you!</span>
```

Comment:
- [ ] Please make this an `h1` tag instead of a `span` tag.  This is the title of your section 😎 and so it is recommended for sections to have `h1` headings 👍